### PR TITLE
chore: update readme to include a features section

### PR DESCRIPTION
**Pull request by beeme1mr**
## This PR

- adds a new banner
- Adds a feature section

### Related Issues

Part of #179 

### Follow-up Tasks

Move docs to the wiki section of the repo. It's easier to maintain and structure.

### How to test

Signed-off-by: Michael Beemer <beeme1mr@users.noreply.github.com>

### CODE REVIEW

**Comment by beeme1mr** [comment]
I would like some feedback on this to see what everyone thinks. I would also like to move most of the docs to [the wiki](https://github.com/open-feature/flagd/wiki) section of the repo. This would allow us to more easily manage and structure the docs.

**Review by AlexsJones — APPROVED** [review]
I like it, lets do it

File: README.md [hunk 1]
```diff
@@ -1,12 +1,35 @@
...
+    <img src="https://img.shields.io/github/release/open-feature/flagd/all.svg">
+  </a>
+</p>
+
+## Features
+
+- OpenFeature compliant with providers available in many languages.
+- Multiple flag configuration sources including `files`, `http`, and `Kubernetes`.
+- Accessible over gRPC and HTTP.
+- Able to subscribe to real-time flag change events.
```
Comment (toddbaert) [inline README.md:25]:
```suggestion
- Supports subscriptions to real-time flag change events.
```

File: README.md [hunk 2]
```diff
@@ -1,12 +1,35 @@
...
+  </a>
+  <img src="https://img.shields.io/github/go-mod/go-version/open-feature/flagd">
+  <a href="https://github.com/open-feature/flagd/releases">
+    <img src="https://img.shields.io/github/release/open-feature/flagd/all.svg">
+  </a>
+</p>
+
+## Features
+
+- OpenFeature compliant with providers available in many languages.
```
Comment (toddbaert) [inline README.md:22]: Since these are fragments and not sentences, please remove the capitalization and periods.

**Comment by toddbaert** [comment]
:fire:

**Comment by beeme1mr** [comment]
I started the wiki but it's taking a bit longer than expected. I'll try and have it wrapped up by tomorrow.

File: README.md (hunk 2 above)
Comment (beeme1mr) [reply README.md:22]: I can remove the punctuation but it feels weird not to capitalize the bullet points.

**Pull request merged**